app/views/layouts/base.html.erb

```diff
@@ -33,11 +33,35 @@
   <![endif]-->
   <%= csrf_meta_tags %>
 
-  <link href="/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180" />
-  <link href="/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png" />
-  <link href="/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png" />
+  <%= tag(
```

Comment:
`favicon_link_tag` assumes the source is in the image folder in the asset pipeline, so we can't use it here
https://github.com/rails/rails/blob/v7.0.4/actionview/lib/action_view/helpers/asset_tag_helper.rb#L280

Reply:
Hm, interesting. I thought this was working for me when I tested locally. I'd also expect the folders to be part of the asset pipeline, since they're configured in the manifest?

https://github.com/18F/identity-idp/blob/a5ca5e4c6cfa90745167039039e5e2af7ed729bb/app/assets/config/manifest.js#L11

Not a big issue either way thogh.

Reply:
Yeah, I messed it up by using `favicon-16x16` instead of `favicon-16`, I reverted to use `favicon_link_tag`